### PR TITLE
Spruce up the course website, including the schedule

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Here I'm adding the links that are ready to the workshop schedule. I use the `{{site.release_tag}}` convention, so once  AlexsLemonade/training-modules#241 and #6 are accomplished this should just work :tm:. I also adding resources for consultation days and guidelines for posting an error in Slack to the navigation bar.

I expect I will be told that `scRNA-seq Exercise 2: Here we go again!` is not an acceptable thing to title the scRNA-seq day 2 exercise notebook and I've made peace with that.

### CODE REVIEW

**Comment by jashapiro** [comment]
This seems to include most of the content of #19... I will check for differences and look to merge in any changes.

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -10,28 +10,45 @@ __*All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt)
...
+| 3:00 PM     | Questions and introduction to the exercise: <br> [Marker genes in single-cell data](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/03-scrnaseq_day1_exercise.Rmd) | Zoom: [Breakout Rooms](../virtual-setup/zoom-procedures.md#using-zoom-breakout-rooms) |
+| 3:30 PM     | [Consultation session](workshop-structure.md#consultation-sessions) begins | [`#2020-july-training` Slack channel](../virtual-setup/slack-procedures.md#general-use) <br> (See [Guidelines for Posting Errors to Slack](posting-errors-guidelines.md)) |
 | 5:00 PM     | End             |
-| **Day 2**   | **2020-07-28** <br> Introduction to Single-Cell RNA-Seq |
-| 12:00 PM    | Quantification of droplet-based scRNA-seq data <br> Dimension reduction | Zoom: Main Session |
-| 3:00 PM     | Questions and introduction to the exercise | Zoom: Breakout Rooms | 
-| 3:30 PM     | Consultation session begins | `#2020-july-training` Slack channel |
+| **Day 2**   | **2020-07-28** <br> [Introduction to Single-Cell RNA-Seq](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/README.md) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:21]:
```suggestion
| **Day 2**   | **2020-07-28** <br> [**_Introduction to Single-Cell RNA-seq_**](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/README.md) |
```

File: workshop/SCHEDULE.md [hunk 2]
```diff
@@ -10,28 +10,45 @@ __*All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt)
 
 | Time        | Topic                                          | Location |
 |-------------|------------------------------------------------|----------|
-| **Day 1**   | **2020-07-27** <br> Introduction to Single-Cell RNA-Seq |  |
-| 12:00 PM    | Welcome, Introductions, and Getting Started <br> [Workshop Intro Slides (pdf)](../slides/2020-07-27_CCDL_Workshop_Intro.pdf) | Zoom: Main Session |
-| 1:00 PM     | Normalization and QC of scRNA-seq data <br> [Intro to Single-Cell RNA-seq (pdf)](../slides/2020-07-27_CCDL_Workshop_Intro.pdf)| Zoom: Main Session |
-| 3:00 PM     | Questions and introduction to the exercise | Zoom: Breakout Rooms
-| 3:30 PM     | Consultation session begins | `#2020-july-training` Slack channel |
+| **Day 1**   | **2020-07-27** <br> [Introduction to Single-Cell RNA-Seq](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/README.md) |  |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:13]:
```suggestion
| **Day 1**   | **2020-07-27** <br> [**_Introduction to Single-Cell RNA-seq_**](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/README.md) |  |
```

File: workshop/SCHEDULE.md [hunk 3]
```diff
@@ -10,28 +10,45 @@ __*All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt)
...
+| 3:00 PM     | Questions and introduction to the exercise | Zoom: [Breakout Rooms](../virtual-setup/zoom-procedures.md#using-zoom-breakout-rooms) | 
+| 3:30 PM     | [Consultation session](workshop-structure.md#consultation-sessions)  begins | [`#2020-july-training` Slack channel](../virtual-setup/slack-procedures.md#general-use) <br> (See [Guidelines for Posting Errors to Slack](posting-errors-guidelines.md)) |
 | 5:00 PM     | End | |
-| **Day 4**   | **2020-07-30** <br> Introduction to Machine Learning | 
-| 12:00 PM    | Clustering and heatmap construction | Zoom: Main Session |
-| 3:00 PM     | Questions and introduction to the exercise | Zoom: Breakout Rooms | 
-| 3:30 PM     | Consultation session begins | `#2020-july-training` Slack channel |
+| **Day 4**   | **2020-07-30** <br> [Introduction to Machine Learning](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/machine-learning/README.md) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:39]:
```suggestion
| **Day 4**   | **2020-07-30** <br> [**_Introduction to Machine Learning_**](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/machine-learning/README.md) | 
```

File: workshop/SCHEDULE.md [hunk 4]
```diff
@@ -10,28 +10,45 @@ __*All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt)
...
+| 1:00 PM     | Break | |
+| 1:15 PM     | [scRNA-seq Dimension reduction](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/05-dimension_reduction_scRNA-seq.nb.html) |  Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
+| 2:45 PM     | Break | | 
+| 3:00 PM     | Questions and introduction to the exercise: <br> [scRNA-seq Exercise 2: Here we go again!](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/06-scrnaseq_day2_exercise.Rmd) | Zoom: [Breakout Rooms](../virtual-setup/zoom-procedures.md#using-zoom-breakout-rooms) | 
+| 3:30 PM     | [Consultation session](workshop-structure.md#consultation-sessions)  begins | [`#2020-july-training` Slack channel](../virtual-setup/slack-procedures.md#general-use) <br> (See [Guidelines for Posting Errors to Slack](posting-errors-guidelines.md)) |
 | 5:00 PM     | End | |
 | **Day 3**   | **2020-07-29** <br> Introduction to Pathway Analysis |   |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:29]:
```suggestion
| **Day 3**   | **2020-07-29** <br> **_Introduction to Pathway Analysis_** |   |
```

File: workshop/SCHEDULE.md [hunk 5]
```diff
@@ -10,28 +10,45 @@ __*All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt)
...
-| 1:00 PM     | Normalization and QC of scRNA-seq data <br> [Intro to Single-Cell RNA-seq (pdf)](../slides/2020-07-27_CCDL_Workshop_Intro.pdf)| Zoom: Main Session |
-| 3:00 PM     | Questions and introduction to the exercise | Zoom: Breakout Rooms
-| 3:30 PM     | Consultation session begins | `#2020-july-training` Slack channel |
+| **Day 1**   | **2020-07-27** <br> [Introduction to Single-Cell RNA-Seq](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/README.md) |  |
+| 12:00 PM    | Welcome, Introductions, and Getting Started <br> [Workshop Intro Slides (pdf)](../slides/2020-07-27_CCDL_Workshop_Intro.pdf) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
+| 1:00 PM     | <br> [Intro to Single-Cell RNA-seq Slides (pdf)](../slides/2020-07-27_Intro_to_scRNA-seq.pdf) <br> [scRNA-seq QC and filtering](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/01-filtering_scRNA-seq.nb.html) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:15]:
```suggestion
| 1:00 PM     | [Intro to Single-Cell RNA-seq Slides (pdf)](../slides/2020-07-27_Intro_to_scRNA-seq.pdf) <br> [scRNA-seq QC and filtering](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/01-filtering_scRNA-seq.nb.html) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```

File: workshop/resources-for-consultation-sessions.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Resources for Consultation Sessions
+nav_title: Resources for Consultation Sessions
```
Comment (jashapiro) [inline workshop/resources-for-consultation-sessions.md:3]:
```suggestion
nav_title: Consultation Resources
```

**Comment by jaclyn-taroni** [comment]
Sorry not sure how I missed #19 :facepalm:

**Comment by jashapiro** [comment]
> Suggestions:
> "Machine Learning: Take a chance on me"
> "Pathway Analysis: Couldn't escape if I wanted to"

I realize I should have also included this option:
"Machine Learning: Again, the German version is a lot clearer about this"

**Pull request merged**